src/test/kotlin/com/atlassian/performance/tools/sshubuntu/api/SudoSshUbuntuImageIT.kt

```diff
@@ -0,0 +1,32 @@
+package com.atlassian.performance.tools.sshubuntu.api
+
+import com.atlassian.performance.tools.sshubuntu.api.docker.execAsResource
+import com.github.dockerjava.core.DockerClientBuilder
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.util.*
+
+class SudoSshUbuntuImageIT {
+
+    @Test
+    fun shouldStartUbuntu() {
+        val docker = DockerClientBuilder.getInstance().build()
+        val osName = docker
+            .createNetworkCmd()
```

Comment:
Is the low-level knowledge about docker networks needed to use the library?

Reply:
In this form it is. Adding network optionality complicated the implementation.

Pros:
* simpler, non-branching implementation
* exposes the consumer to the real complexity of networks: internal/external addresses/ports - necessary context when dealing with browsers connecting to servers (VU tests) or servers polling themselves locally (infra upgrade REST polling)

Cons:
* it's extra complexity for the users who do not care about connecting multiple containers

Reply:
We definitely have to leave the option for networks, but we might invest into making the usage easier for simpler use cases. I'm not sure if it's worth it. I used this code in multiple tests already and the network creation was easily hidden by a local reused method. Perhaps we should wait and see if it's actually a problem.